### DISCUSSION TITLE
绑定域名后，原ip+端口访问形式失效

### DISCUSSION THREAD

**Discussion by fromyb** [1Panel-dev/1Panel#5063] [Q&A]
### 1Panel 版本

v1.10.6-lts

### 问题描述

开发时用 （A）ip+端口（非80）的形式进行开发上线后，
通过新建域名绑定（B）域名 + 80 端口，
结果原 A 方法失效了

### 重现步骤

1. 新建网站主域名使用 ip+端口
2. 新增域名绑定域名+端口

### 期待的正确结果

面板里新建域名后，多个均生效

**Comment by zhengkunwang223** [comment]
A方法是创建了网站吗？还是说只是访问了ip加应用端口

**Comment by fromyb** [comment]
> A方法是创建了网站吗？还是说只是访问了ip加应用端口

创建网站的时候主域名使用的 0.0.0.0:8080 类似的 ip+端口
